examples/map/map_contouring.py

```diff
@@ -17,11 +18,19 @@
 aiamap = sunpy.map.Map(AIA_193_IMAGE)
 
 ###############################################################################
-# Now find the contours
-contours = aiamap.contour(50000)
+# In find a set of contours, we have to provide the level to contour in the
+# same units as the map data. To find out the units we can inspect
+# `sunpy.map.GenericMap.unit`.
+print(aiamap.unit)
+
+###############################################################################
+# We can see that the units of this map are ``ct``, or counts. We can now
+# chose a contour level, and use the :meth:`~sunpy.map.GenericMap.contour`
+# method to extract the contours.
+contours = aiamap.contour(50000 * u.ct)
```

Comment:
```suggestion
contours = aiamap.contour(50000 * aiamap.unit)
```

I understand if you'd rather be explicit that this is in units of `ct`.

Reply:
Yeah, I think it's worth keeping it explicit here.